root/utility/groupRelationships.js

```diff
@@ -13,8 +13,8 @@ import {interpolate, interpolateText}
   from '../static/scripts/edit/utility/linkPhrase';
 
 export type GroupedRelationshipsT = {
-  [CoreEntityTypeT]: {
-    [string]: {
+  [entityType: CoreEntityTypeT]: {
+    [textPhrase: string]: {
```

Comment:
`textPhrase` can be found at:
https://github.com/metabrainz/musicbrainz-server/blob/067a23dbe4893b9e37b5174372bd2f61952c4f5e/root/utility/groupRelationships.js#L47

https://github.com/metabrainz/musicbrainz-server/blob/067a23dbe4893b9e37b5174372bd2f61952c4f5e/root/utility/groupRelationships.js#L119